pkg/retry/errors.go

```diff
@@ -82,7 +83,7 @@ var (
 func IsConnectionError(err error) bool {
 	err = errors.Cause(err)
 	switch err {
-	case driver.ErrBadConn, tmysql.ErrBadConn, gmysql.ErrBadConn:
+	case driver.ErrBadConn, tmysql.ErrBadConn, gmysql.ErrBadConn, mysql.ErrInvalidConn:
```

Comment:
we may not retry the `ErrInvalidConn` when it came from `txn.Commit()`

Reply:
In the situation, do the other three error can safely retry?

Reply:
ok

Reply:
Currently, we will ignore invalid connection error for some DDL. https://github.com/pingcap/dm/blob/fadd54342759286e7572bb100684fe48942c0ca7/syncer/error.go#L79